products/llbuildSwift/BuildSystemBindings.swift

```diff
@@ -581,6 +581,9 @@ public final class BuildSystem {
             _delegate.fs_get_file_info = { BuildSystem.toSystem($0!).fsGetFileInfo(String(cString: $1!), $2!) }
             // FIXME: This should be a separate callback, not shared with getFileInfo (or get FileInfo should take a parameter).
             _delegate.fs_get_link_info = { BuildSystem.toSystem($0!).fsGetFileInfo(String(cString: $1!), $2!) }
+
+            // FIXME: should support fs_create_symlink, but for now explicitly defers to built-in symlink
+            _delegate.fs_create_symlink = nil
```

Comment:
Any reason not to add this in now?

Reply:
Well, right now the FileSystem protocol in the swift bindings is pretty paltry.  It is fudging the getLinkInfo call by using getFileInfo (🤢).  The comment on BuildSystemBindings.swift:368 mentions a desire to sit on top of the protocol from SwiftPM.   Probably makes sense just clean the whole thing up when we can sit on TSC.

Reply:
OK!